packages/aila/src/utils/lessonPlan/fetchLessonPlanContentById.ts

```diff
@@ -18,8 +19,24 @@ export async function fetchLessonPlanContentById(
   if (!lessonPlanRecord) {
     return null;
   }
+
+  const upgradeResult = await upgradeQuizzes({
```

Comment:
Would be quite nice to have a rolling updates pipeline which is always called ... though mostly will be a trivial noop.

Reply:
Yep, I agree. I think we can start to think about that as we start generalising the schema updates. eg: quizv3 gives us more quiz schema versions, and then later we'll have more fields using schemas